src/patternfly/components/Button/examples/button-states-example.hbs

```diff
@@ -100,3 +100,19 @@
 {{#> button button--modifier="pf-m-link pf-m-inline" button--attribute="disabled"}}
   Inline link button
 {{/button}}
+<br><br>
+{{#> button button--modifier="pf-m-control"}}
+  Control button
```

Comment:
Do you think this variations and "link" and "inline link" above need the word "button" in the examples?

Reply:
No I think its redundant, updated all the examples to remove button